packages/react-native-codegen/src/parsers/parser.js

```diff
@@ -414,4 +414,11 @@ export interface Parser {
    * @returns: the properties of an object represented by a type annotation.
    */
   getObjectProperties(typeAnnotation: $FlowFixMe): $FlowFixMe;
+
+  /**
+   * Given a typeAnnotation return the properties of an object.
```

Comment:
I imagine that's a copypaste?.. 

Could you please modify this so it actually does correspond to what the method does? :)

Reply:
oops, I forgot to update that 1st line, will fix! Thanks for the review :)

Reply:
updated the description @rshest